Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/validation/core/DccValidation.kt

```diff
@@ -13,6 +13,8 @@ import org.joda.time.LocalDateTime
 data class DccValidation(
     val userInput: ValidationUserInput,
     val validatedAt: Instant,
+    val signatureCheckPassed: Boolean,
+    val certificateExpiredAt: LocalDateTime,
```

Comment:
I would have passed the Instant here and converted in the UI. But that's up to you

Reply:
Actually, I would have taken the date from the certificate, and not have added it to the validation result.

Reply:
You are right, totally forgot that, it should be aligned with other related certificate information in other view holders